kmp-nativecoroutines-compiler/src/main/kotlin/com/rickclephas/kmp/nativecoroutines/compiler/KmpNativeCoroutinesSyntheticResolveExtension.kt

```diff
@@ -109,7 +109,8 @@ internal class KmpNativeCoroutinesSyntheticResolveExtension(
         val type = thisDescriptor.module.getExpandedNativeFlowType(valueType)
         return createPropertyDescriptor(thisDescriptor, coroutinesPropertyDescriptor.visibility,
             name, type, coroutinesPropertyDescriptor.dispatchReceiverParameter,
-            coroutinesPropertyDescriptor.extensionReceiverParameter
+            coroutinesPropertyDescriptor.extensionReceiverParameter,
+            coroutinesPropertyDescriptor.contextReceiverParameters
```

Comment:
Support should also be added to the IRTransformer

Reply:
Turns out these are part of the `valueParameters` so no changes are required in the IrTransformer